app/models/spotlight/resources/searchworks.rb

```diff
@@ -1,15 +1,14 @@
 module Spotlight::Resources
+  # Resource provider for searchworks pages
   class Searchworks < Spotlight::Resources::DorResource
-
     self.weight = -1000
 
-    def self.can_provide? res
-      !!(res.url =~ /^https?:\/\/searchworks[^\.]*.stanford.edu/)
+    def self.can_provide?(res)
+      res.url.match(%r{^https?://searchworks[^\.]*.stanford.edu/}).present?
     end
 
     def doc_id
-      url.match(/^https?:\/\/searchworks[^\.]*.stanford.edu\/.*view\/([^\/\.#]+)/)[1]
+      url.match(%r{^https?://searchworks[^\.]*.stanford.edu/.*view/([^/\.#]+)})[1]
```

Comment:
bleah.  I dislike %r  as I have been imprinted with  /your-regex-here/, and %r  seems a bit precious to Ruby.  IMO, it makes the code LESS readable.